messages/create.json

```diff
@@ -8,7 +8,8 @@
   ],
   "flags": {
     "alias": "set an alias for the created username to reference within the CLI",
-    "definitionfile": "file path to a user definition"
+    "definitionfile": "file path to a user definition",
+    "setpassedusernameunique": "if a username is passed, set it unique by appending the org id"
```

Comment:
Because "Username" in the definition file is required (right?) I think it's redundant to include "passed" in the flag name.  so we could simplify a bit. How about "--forceuniqueusername".  Or "--setuniqueusername". 

Do we have a short name?  We definitely don't need one, it's okay to just have long name. But if we do define one, how about -q?  We can't use -f or -u or -s, and q is a strong consonant in "unique".   We use -q a lot for query-related parameters in other commands, would that be confusing?  I don't think so, but let me know. 

My suggested edit of description: "force a unique username by appending the org ID to the username defined in the definition file"

Reply:
@jshackell-sfdc Actually username isn't mandatory. If we don't pass one either in the file or in the varargs, it generates ones with the default username + a timestamp.

I thought it would be better to add the id that makes the username unique (org specific) only for passed username, as the generated one is already unique.

I made the tests that way, but I can definitely change it if you think this doesn't make sense.

I do agree though that the flag name is a little bit long but I fell short on a better idea.

Reply:
Ooops, yep, you're right that we generate a username if it's not passed!  Forgot about that, now it makes sense that you want "passed" in the name.  Let me rethink, see if I can suggest a shorter name.  Not that it HAS to be short, long long names are fine too.  Thanks for clarifying.

Reply:
I like `--setuniqueusername` too, just what do you think the behavior should be for an automatically generated username ?
Maybe using `--setuniqueusername` but write in the flag description that this is on passed username only ?
That could be a good compromise.

Reply:
Yes, I think that's a good compromise.  Maybe something like "force the username, if specified in the definition file or at the command line, to be unique by appending the org ID."     Our style is to start the descriptions with a verb if possible, which is why I put the if statement in the middle.

Reply:
Ok I'll got for it then, thanks ! Is there a guideline about styles we should respect, like starting with a verb ?

Reply:
@jshackell-sfdc I've updated the PR with what we said.

Reply:
Our style guide is here: https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_plugins.meta/sfdx_cli_plugins/cli_plugins_style_descriptions.htm